Src/Web/Web.Shared.Net/RequestTrackingTelemetryModule.cs

```diff
@@ -23,7 +23,7 @@
     public class RequestTrackingTelemetryModule : ITelemetryModule, IDisposable
     {
         private readonly EventListener listener;
-
+        private readonly IList<FilterRequest> handlersToFilter = new List<FilterRequest>();
```

Comment:
I couldn't get it to work with the XML config. Do you know how to use a list of strings for XML config?